degica/komoju-woocommerce
# Add option to use 'on-hold' status on payment authorization

User request.

This adds an option to use the 'on-hold' status for authorized payments. This allows inventory to be reserved in the case where eg. a konbini or bank transfer payment is approved but awaiting payment. This is added as an option as some merchants may wish to continue operating with the old behavior.

## How to test

1. Visit the settings screen under Woocommerce -> payments and confirm that the new option to use on-hold status appears
2. Select the option and make a konbini payment
3. Confirm the status of the order in the Woocommerce dashboard is "on-hold"
4. Untick the option and repeat 3
5. Confirm the status of the order is "payment pending"
